### DISCUSSION TITLE
Media sources and paths

### DISCUSSION THREAD

**Discussion by jschlyter** [nimroddolev/chime_tts#192] [Q&A]
I have defined a media directory in HASS as follows:
 
```
homeassistant:
  media_dirs:
    chime_tts: /media/chime_tts
```

Chime TTS _Media folder_ is set to `chime_tts` and the temporary directory to `/media/chime_tts/tmp/`. When playing audio, a media-source with URI path `/media/chime_tts/chime_tts/tmp/xgr8o98b.mp3` is generated and playout fails.

There seems to be some magic in `get_media_content_id`, but I fail to understand how this is supposed to work at all unless you use very specific names for the media sources and paths.

In my specific setup, adding a soft link from `chime_tts` to `.` works for now.

**Comment by jschlyter** [comment]
Any ideas @nimroddolev ?

**Reply by nimroddolev** [reply]
Hi @jschlyter. I’m sorry you’re having this issue.

To help me get a better idea of what might be happening, can you please complete the following?

1. Upgrade to v1.1.5
2. Enable [debug logging](https://nimroddolev.github.io/chime_tts/docs/support#debug-logging)
3. Call `chime_tts.say` again
4. Provide me with the debug log messages

**Comment by GerhardKreuzer** [comment]
Hi,
same problem here.
Maybe you can add some documentation, how we should use this feature.
I tried to add a file in /meda/sounds, I add this path also to my ChimeTTS configuration, but cant find the sound in the list. Its a wav file, is this the issue?
I think, some mor documentaionen and a simple step by step guide will help a lot.

Thanks.
With best regards
Geerhard
PS: Rest works great for now, only installation is little bit strange.
![image](https://github.com/user-attachments/assets/ce65c1f7-5909-40d6-b0a1-500b5ada3174)

**Reply by nimroddolev** [reply]
Hi @jschlyter and @GerhardKreuzer.

please review the documentation for [configuring your custom chimes folder](https://nimroddolev.github.io/chime_tts/docs/documentation/actions/say-action/#custom-local-audio-files-and-external-urls).

if you are still having issues, can you please:
1. Uninstall and reinstall the latest version
2. Provide a screenshot of your [configuration](https://my.home-assistant.io/redirect/integration/?domain=chime_tts)

**Reply by GerhardKreuzer** [reply]
Hi Nim,
just installed ChimeTTS, so it should be the newest version.

Here my config:

action: chime_tts.say
metadata: {}
data:
  tts_platform: tts.piper
  tts_speed: 91
  cache: false
  announce: false
  message: Es ist nun 11 Uhr.
  chime_path: **/media/sounds/windgong.wav**
target:
  device_id: 40896b4023a6e26dbf06ec0fc889f7a1

And here is the file:
![grafik](https://github.com/user-attachments/assets/bc1886d2-f7c0-421f-b6f4-a9abdd37fecf)

No sound is playing but the text is spoken.

With best regards

Gerhard

**Reply by nimroddolev** [reply]
I believe the issue is that the audio filename is in all capitals, whereas the filename in the YAML you provided is all lowercase. Please match the filename exactly and try again.
